commons/src/main/java/io/aiven/kafka/connect/common/config/FileNameFragment.java

```diff
@@ -40,7 +40,7 @@ public final class FileNameFragment extends ConfigFragment {
 
     // package private so that testing can access.
     static final String GROUP_FILE = "File";
-    static final String FILE_COMPRESSION_TYPE_CONFIG = "file.compression.type";
+    public static final String FILE_COMPRESSION_TYPE_CONFIG = "file.compression.type";
```

Comment:
The `FILE_COMPRESSION_TYPE_CONFIG = "file.compression.type"` is defined in multiple places. Some cleanup and maybe using `commons/src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java` would be preferred?
Comment applies also to `file.name.template` and looks that alignment is something to work on a separate PR.